scripts/giphy.py

```diff
@@ -0,0 +1,55 @@
+"""
+Respond to:
+
+    /giphy "phrase"
+
+with a random image from giphy based on that phrase.
+"""
+
+import asyncio
+import functools
+import hangups
+import logging
+import re
+import requests
+import urllib
+
+
+logger = logging.getLogger('hangbot.scripts.giphy')
+
+
+API_KEY = 'dc6zaTOxFJmzC'  # Use the public API key for now
+API_URL = 'http://api.giphy.com/v1/gifs/random'
+COMMAND = '/giphy '
+MESSAGE_REGEX = re.compile(r'^{}'.format(COMMAND), re.IGNORECASE)
```

Comment:
Oh, and do you _really_ want to make it case-insensitive? Up to you, but just pointing that out.